### DISCUSSION TITLE
<title>Can't able to create Organization

### DISCUSSION THREAD

**Discussion by Priyankagunturu** [community/community#44544] [Repositories]
### Select Topic Area

Bug

### Body

When i tried to create a new organization it throws me captcha error. I have followed all troubleshooting steps but doesn't even cross checked with the below as per github team mentioned. 

If you're unable to verify with the captcha:

Ensure JavaScript is enabled on your browser.
Ensure your browser is supported. If your browser isn't supported, upgrade your browser or install a supported browser. For a list of supported browsers, see "[Supported browsers](https://docs.github.com/en/articles/supported-browsers)."
Ensure your network configuration is not blocking https://octocaptcha.com/ or https://arkoselabs.com/. If you're behind a corporate firewall, contact your IT administrator to allow those domains. To verify access to these domains, visit https://octocaptcha.com/test and ensure the text "Connection successfully made!" is displayed, then visit https://client-demo.arkoselabs.com/github and ensure you are able to load the captcha.
Ensure your browser does not have plug-ins or extensions that may be interfering with GitHub. If so, temporarily disable the plug-ins or extensions during captcha verification.

![Screenshot (373)](https://user-images.githubusercontent.com/94977766/212811910-bcd362e8-56d5-45bb-a5f6-47a8dc810412.png)

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @Priyankagunturu! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, there are a few troubleshooting steps I can suggest:
- What browser are you using? For the best experience with GitHub, we recommend using the latest version of [Chrome](https://google.com/chrome), [Edge](https://www.microsoft.com/en-us/edge), [Firefox](https://mozilla.org/firefox), or [Safari](https://apple.com/safari).
- Can you try this again using your browser's private browsing/incognito mode? Here's an article on how to enable that for each of these browsers: [How to go incognito in Chrome, Edge, Firefox, and Safari](https://www.computerworld.com/article/3356840/how-to-go-incognito-in-chrome-firefox-safari-and-edge.html).
- Just in case this is a network related issue, can you try connecting to another internet source to see if that resolves the issue for you? 

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by Souvik2376** [comment]
@ghostinhershell Same issue can't create organization on Free (The basics for individuals and organizations) because of the new verification . I tried many times the new captcha is not good at all. The puzzle and music captcha both have some bug .

**Reply by elismasilva** [reply]
i tried today its not working yet

**Comment by S1ngularity96** [comment]
I have the same problem

**Reply by gepp4** [reply]
I have the same problem I tried yesterday and today for at least 15m.

**Comment by ghostinhershell** [comment]
Thanks so much for following up, I see that this is an issue that a couple of people are running into. I tested this out on my end and I was able to successfully create an organization. The only thing to note is that I did get the CAPTCHA incorrect on my first try. It seemed like a subtle difference between the direction of the fingers and how the image was faced. Here's an example of one I got correct:
![image](https://github.com/community/community/assets/14021469/39cebb7c-fba0-4bac-af84-4cfcb8ccbd7b)

Can you guys try doing this again in incognito mode ([or your browser's equivalent](https://www.computerworld.com/article/3356840/how-to-go-incognito-in-chrome-firefox-safari-and-edge.html)) and let me know if you guys are still running into this issue?

**Reply by gepp4** [reply]
I tried with Chrome in incognito mode and it worked. Thanks!

**Reply by GeminiNinth** [reply]
Still not working even in incognito mode.

**Reply by GeminiNinth** [reply]
I did this on my desktop PC at home and it didn't solve anything, but when I was on the road working on my laptop, I was able to verify without any problems. This may not be much information, but I'm sharing it just in case.

**Reply by n-chouhan** [reply]
its a problem still i cant through either

**Reply by the-anshu-dev** [reply]
try once on incognito mode, may be browser cache creates some problem
